openmdao/solvers/ln_direct.py

```diff
@@ -8,7 +8,22 @@
 
 class DirectSolver(ScipyGMRES):
     """ OpenMDAO LinearSolver that explicitly solves the linear system using
-    linalg.solve."""
+    linalg.solve.
+
+    Options
+    -------
+    options['atol'] :  float(1e-12)
```

Comment:
wonder if this would be more user friendly if we strip the "options['']" wrapper and just show the option name